Index: common/core/src/main/java/com/retrowalk/models/AddressRequest.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/common/core/src/main/java/com/retrowalk/models/AddressRequest.java b/common/auth-app/src/main/java/com/retrowalk/models/request/AddressRequest.java
rename from common/core/src/main/java/com/retrowalk/models/AddressRequest.java
rename to common/auth-app/src/main/java/com/retrowalk/models/request/AddressRequest.java
--- a/common/core/src/main/java/com/retrowalk/models/AddressRequest.java	(revision 9a892109e0a8eade00e41397f441e558fa09f401)
+++ b/common/auth-app/src/main/java/com/retrowalk/models/request/AddressRequest.java	(date 1728911925900)
@@ -1,4 +1,4 @@
-package com.retrowalk.models;
+package com.retrowalk.models.request;
 
 import lombok.AllArgsConstructor;
 import lombok.Builder;
